src/scancode/api.py

```diff
@@ -105,30 +105,42 @@ def get_emails(location):
     """
     Yield mappings of emails detected in the file at `location`.
     """
+    email_threshold  = 100 
+    email_count = 0
     from cluecode.finder import find_emails
     for email, line_num  in find_emails(location):
         if not email:
             continue
+        email_count += 1
         misc = OrderedDict()
         misc['email'] = email
         misc['start_line'] = line_num
         misc['end_line'] = line_num
-        yield misc
+        if email_count < 100:
+            yield misc
+    if email_count >= 100:
+        print("\nWarning : %d emails detected at %s but only %d were reported"%(email_count,location,email_threshold))
```

Comment:
As said in the ticket no warning neeed

Reply:
So for now i'll just remove the warning and break while the hardcoded threshold is reached ?

Reply:
yes, and use enumerate with the threshold as an arg with a default to 50